app/views/content/explore-teaching-advisers.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Explore teaching advisers
+description: |-
+    Find out if teaching is for you with an explore teaching adviser who can help you find out what teaching is really like.
```

Comment:
this is a little bit repetitive - find out / help you find out

very minor but worth rewording slightly?